packages/p2p/src/components/my-profile/my-profile.jsx

```diff
@@ -73,46 +79,78 @@ const MyProfile = observer(() => {
                 <React.Fragment>
                     <Table>
                         <Table.Row className='my-profile__stats'>
-                            <div className='my-profile__stats-cell-separator' />
-                            <Table.Cell className='my-profile__stats-cell'>
-                                <div className='my-profile__stats-cell-header'>{localize('Total orders')}</div>
-                                <div className='my-profile__stats-cell-info'>{total_orders_count || '-'}</div>
-                            </Table.Cell>
                             <div className='my-profile__stats-cell-separator' />
                             <Table.Cell className='my-profile__stats-cell'>
                                 <div className='my-profile__stats-cell-header'>
-                                    {localize('Buy ({{currency}})', { currency: general_store.client.currency })}
+                                    <Text size={isMobile() ? 'xxxs' : 's'} color='gray'>
+                                        {localize('Total orders')}
+                                    </Text>
                                 </div>
-                                <div className='my-profile__stats-cell-info'>{buy_orders_count || '-'}</div>
+                                <div className='my-profile__stats-cell-info'>{total_orders_count || '-'}</div>
                             </Table.Cell>
                             <div className='my-profile__stats-cell-separator' />
-                            <Table.Cell className='my-profile__stats-cell'>
-                                <div className='my-profile__stats-cell-header'>
-                                    {localize('Sell ({{currency}})', { currency: general_store.client.currency })}
-                                </div>
-                                <div className='my-profile__stats-cell-info'>{sell_orders_count || '-'}</div>
-                            </Table.Cell>
+                            {isMobile() ? (
+                                <Table.Cell className='my-profile__stats-cell'>
+                                    <div className='my-profile__stats-cell-header'>
+                                        <Text size={isMobile() ? 'xxxs' : 's'} color='gray'>
```

Comment:
Can we use the themes for the color? So as per design this color would be `--text-less-prominent`, can we apply the same as color to the Text component.